e2e/tests/navigation.test.ts

```diff
@@ -0,0 +1,55 @@
+/*
+MIT License
+
+Copyright (c) 2021-present, Elastic NV
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in
+all copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+THE SOFTWARE.
+*/
+
+import { ElectronServiceFactory, TestBrowserService } from "../services";
+
+const electronService = new ElectronServiceFactory();
+
+afterEach(() => {
+  electronService.terminate();
+});
+
+describe("Navigation", () => {
+  it("records chromium's opened pages", async () => {
+    const electronWindow = await electronService.getWindow();
+
+    await electronWindow.type(
+      '[placeholder="Enter URL to test"]',
+      "https://elastic.co/"
+    );
+
+    await electronWindow.click("text=Start recording");
+    const remoteBrowserPage = await TestBrowserService.getRemoteBrowserPage();
```

Comment:
Great idea! This was an excellent comment because it also highlighted the importance of encapsulating these actions into the electron service itself, in kind of like a [`PageObject`](https://martinfowler.com/bliki/PageObject.html), which I just did.

Thank you!